pkg/gotohelm/gochart.go

```diff
@@ -58,7 +77,17 @@ func (c *GoChart) LoadValues(values any) (helmette.Values, error) {
 		return nil, errors.WithStack(err)
 	}
 
-	merged, err := helm.MergeYAMLValues("", c.defaultValues, valuesYaml)
+	merged, err := c.calculateDepsValues()
```

Comment:
I feel that splitting this logic across two functions makes it more difficult to follow. Why not make LoadValues itself recursive?

Reply:
I inlined that function.